<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBottle</class>
 <widget class="QDialog" name="NewBottle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Bottle Tag</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Tag Detected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tag UID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="uidLabel">
       <property name="text">
        <string>[UID]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Bottle Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Max Volume (mL):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="maxVolLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="inputStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="placeholderPage"/>
     <widget class="OnScreenKeyboard" name="keyboardWidget"/>
     <widget class="OnScreenKeypad" name="keypadWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OnScreenKeyboard</class>
   <extends>QWidget</extends>
   <header>onscreenkeyboard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OnScreenKeypad</class>
   <extends>QWidget</extends>
   <header>onscreenkeypad.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
